Announcement re: Sheila

jpstroop opened the pull request:
http://copy_announcement.iiif.io/

zimeon commented:
@jpstroop's update looks good to me :+1:. I don't really like the title of the press release "IIIF hires first staff position to expand the platform's reach and community engagement" because I don't think it makes sense to talk about IIIF as a platform, but I suppose that was what was released so we should just keep it.

azaroth42 commented:
The text is technically copyright Stanford. So we shouldn't merge until we get permission. :P

mikeapp commented on source/_posts/2016-08-05-community-manager.md:
```diff
@@ -0,0 +1,21 @@
+
+Sheila Rabun has been appointed the new Community and Communications Officer of the International Image Interoperability Framework (IIIF) community, and will assume her responsibilities at the end of August.
+
+The IIIF community is a growing membership of the world's leading research libraries and image repositories that have embarked on an effort to collaboratively produce an interoperable technology and community framework for image delivery. Rabun, is the first dedicated staff position for the global community, and will focus on coordinating the expanding suite of IIIF features and functionality, fa ...
```
Stray comma?  `Rabun, is`

azaroth42 replied:
"Platform" ?

azaroth42 commented on source/_posts/2016-08-05-community-manager.md:
```diff
@@ -0,0 +1,21 @@
+
+Rabun will serve as an employee of the Council on Library Information Resources (CLIR) under terms of an agreement with the [IIIF Consortium](http://iiif.io/news/2015/06/17/iiif-consortium/). Rabun will work at the direction of Michael Keller, university librarian at Stanford and Tom Cramer, assistant university librarian and chief technology strategist at Stanford. 
+
+Stanford Libraries, together with the Bodleian Libraries at Oxford University and the British Library conceived the IIIF initiative on the back of a napkin over a dinner conversation, and in June 2015 eight additional institutions came together to establish the IIIF-Consortium.
```
It was 11, no? not eight? Per:  http://iiif.io/news/2015/06/17/iiif-consortium/

tomcramer replied:
3 + 8 = 11.

azaroth42 commented on source/_posts/2016-08-05-community-manager.md:
```diff
@@ -0,0 +1,21 @@
+
+Stanford Libraries, together with the Bodleian Libraries at Oxford University and the British Library conceived the IIIF initiative on the back of a napkin over a dinner conversation, and in June 2015 eight additional institutions came together to establish the IIIF-Consortium.
+
+"Sheila's impressive accomplishments in various digital library and scholarly programs coupled with her strong critical thinking, analysis and communication skills will add great capacity for the entire IIIF community," said Keller, who also serves as the chair of the IIIF-Consortium. "We look forward to partnering with her to shape this very new, and exciting role in support of improved access t ...
```
Inconsistent spelling for the Consortium. Dash or no dash?  (I think no dash)

azaroth42 replied:
"Partnering"? Who is the "we" that is "partnering" ?

tomcramer commented:
Please do NOT merge the text as is. It is not well suited for the IIIF.io site. 

Please use this instead (sorry for loss of markdown): 

Title: IIIF Community & Communications Officer Hired
--------------
Sheila Rabun has been appointed the new Community and Communications Officer of the International Image Interoperability Framework (IIIF) community, and will assume her responsibilities at the end of August, 2016.

IIIF has seen remarkable growth since June 2015, when 11 institutions joined together to formally organize the IIIF Consortium, and support the growing number of specifications, compatible software, interoperable institutions, and community members. Rabun’s is the first dedicated staff position for the global community, and will focus on coordinating the expanding suite of IIIF features and functionality, facilitating communication within and beyond the current community, and helping implementers and individual users engage in the Framework. 

“I am excited to join the IIIF community and build upon the excellent accomplishments of the initiative thus far,” Rabun said in a written statement. Formerly the Digital Project Manager and Interim Director of the University of Oregon Libraries’ Digital Scholarship Center, Rabun managed the development of the Oregon Digital Hydra repository, the Oregon Digital Newspaper Program (ODNP) and Historic Oregon Newspapers online, the collaborative Open Online Newspaper Initiative (Open ONI) software development project, and a variety of digital scholarship projects.

“I am passionate about digital literacy and encouraging the use of digital applications to conduct research, analysis, and dissemination as an extension of traditional academia, and I look forward to strengthening internal and external communication within the IIIF Consortium and broader community, organizing documentation and training efforts, and building new partnerships through increased outreach and advocacy,” said Rabun.

Rabun will serve as an employee of the Council on Library Information Resources (CLIR) under terms of an agreement with the IIIF Consortium. Rabun will work at the direction of Michael Keller, chair of the IIIF Consortium Executive Committee, and Tom Cramer, founder and chair of the IIIF Coordinating Committee.

“Sheila’s impressive accomplishments in various digital library and scholarly programs coupled with her strong critical thinking, analysis and communication skills will add great capacity for the entire IIIF community,” said Keller, who is also the University Librarian at Stanford University. “We look forward to partnering with her to shape this very new, and exciting role in support of improved access to all our digital collections.”

The Community & Communicatoins Officer role is made possible with funding from the IIIF Consortium (IIIF-C), nearly three dozen institutions that have banded together to provide financial, logistical, technological support and content to advance IIIF. If you would like to join IIIF-C as a Founding Member, please email iiif-secretariat@googlegroups.com.

jpstroop commented:
I changed:

> suite of compatible software

to "IIIF-compatible software". To me, "suite" sounds like a closed group of stuff we own, none of which is true.

I also moved the first "(IIIF-C)" up to the first mention of "IIIF Consortium"  And added the links.

jpstroop commented on source/_posts/2016-08-05-community-manager.md:
```diff
@@ -0,0 +1,29 @@
+Rabun will serve as an employee of the Council on Library Information Resources (CLIR) under terms of an agreement with the IIIF Consortium. She will work at the direction of Michael Keller, chair of the IIIF Consortium Executive Committee, and Tom Cramer, founder and chair of the IIIF Coordinating Committee.
+
+"Sheila's impressive accomplishments in various digital library and scholarly programs coupled with her strong critical thinking, analysis and communication skills will add great capacity for the entire IIIF community," said Keller, who is also the University Librarian at Stanford University. "We look forward to partnering with her to shape this very new, and exciting role in support of improved  ...
+
```
Agree that "partnering" seems like a strange word to use, but it's a quote, so...

jpstroop commented on source/_posts/2016-08-05-community-manager.md:
```diff
@@ -0,0 +1,29 @@
+
+Rabun will serve as an employee of the Council on Library Information Resources (CLIR) under terms of an agreement with the IIIF Consortium. She will work at the direction of Michael Keller, chair of the IIIF Consortium Executive Committee, and Tom Cramer, founder and chair of the IIIF Coordinating Committee.
+
+"Sheila's impressive accomplishments in various digital library and scholarly programs coupled with her strong critical thinking, analysis and communication skills will add great capacity for the entire IIIF community," said Keller, who is also the University Librarian at Stanford University. "We look forward to partnering with her to shape this very new, and exciting role in support of improved  ...
```
Agree that "partnering" seems like a strange word to use, but it's a quote, so...

azaroth42 commented:
Good to merge?  You can see live version at:
    http://copy_announcement.iiif.io/news/2016/08/05/community-manager/

(I still don't like "partnering", but now it's clearer that it's Stanford. Sort of.  Prepared to hold nose.)

mikeapp commented:
👍

azaroth42 commented:
Errors are stupid link checker. Merging from Mike's +1.
